Mathlib/Analysis/Fourier/FourierTransformDeriv.lean

```diff
@@ -106,11 +106,11 @@ theorem hasFDerivAt_fourier [CompleteSpace E] [MeasurableSpace V] [BorelSpace V]
         L.continuous.aestronglyMeasurable.prod_mk hf.1
       apply aux0.comp_aestronglyMeasurable aux1
   have h4 : (∀ᵐ v ∂μ, ∀ (w' : W), w' ∈ Metric.ball w 1 → ‖F' w' v‖ ≤ B v) := by
-    refine ae_of_all _ (fun v w' _ ↦ ?_)
+    filter_upwards with v w' _
     exact norm_fderiv_fourier_transform_integrand_right_le L f v w'
   have h5 : Integrable B μ := by simpa only [← mul_assoc] using hf'.const_mul (2 * π * ‖L‖)
-  have h6 : ∀ᵐ v ∂μ, ∀ w', w' ∈ Metric.ball w 1 → HasFDerivAt (fun x ↦ F x v) (F' w' v) w' :=
-    ae_of_all _ (fun v w' _ ↦ hasFDerivAt_fourier_transform_integrand_right L f v w')
+  have h6 : ∀ᵐ v ∂μ, ∀ w', w' ∈ Metric.ball w 1 → HasFDerivAt (fun x ↦ F x v) (F' w' v) w' := by
+    filter_upwards with v w' _ using hasFDerivAt_fourier_transform_integrand_right L f v w'
```

Comment:
here the `filter_upwards` version is longer (and presumably also slower) than the original